### ISSUE TITLE
issue with PSR2.Methods.FunctionCallSignature.SpaceBeforeOpenBracket

### ISSUE THREAD

**Issue by yannickl88** [squizlabs/PHP_CodeSniffer#1678]
In the case the } are followed by parenthesis the sniffer will give a false positive. For example:
```php
<?php
function foo() {

}
(count([])) ? true : false;
```

will result in the following errors:
```
 2 | ERROR   | [x] Opening brace should be on a new line (Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine)
 4 | ERROR   | [x] Function closing brace must go on the next line following the body; found 1 blank lines before brace (PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose)
 4 | ERROR   | [x] Space before opening parenthesis of function call prohibited (PSR2.Methods.FunctionCallSignature.SpaceBeforeOpenBracket)
 4 | ERROR   | [x] Opening parenthesis of a multi-line function call must be the last content on the line (PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket)
 5 | ERROR   | [x] Closing parenthesis of a multi-line function call must be on a line by itself (PSR2.Methods.FunctionCallSignature.CloseBracketLine)
```

The fixer will even make this of it:
```php
<?php
function foo()
{
}(count([])) ? true : false;
```

This is I think not the intended use case.

**Comment by yannickl88** [comment]
Another use case where this goes wrong is when an `if` is followed by something in parenthesis. Like so:
```php
if (true) {
}
(new \RuntimeException('foobar'))->getMessage();
```

**Comment by jrfnl** [comment]
Duplicate of #1669 ? If so, this has been fixed in `master` already.

**Comment by yannickl88** [comment]
look like it, did a search and could not find it. Still no release where this is fixed though.
